### PR TITLE
zh-cn: fix the markdown rendering error and improve translation

### PR DESCRIPTION
**Pull request by kiner-tang**
### Description

Before:

image

After:

image

### CODE REVIEW

File: files/zh-cn/web/css/scrollbar-color/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ slug: Web/CSS/scrollbar-color
 
 {{CSSRef}}{{SeeCompatTable}}
 
-**`scrollbar-color`**该[CSS](/zh-CN/docs/Web/CSS)属性设置滚动条轨道和拇指的颜色
+**`scrollbar-color`** 该[CSS](/zh-CN/docs/Web/CSS)属性设置滚动条轨道和拇指的颜色
```
Comment (yin1999) [inline files/zh-cn/web/css/scrollbar-color/index.md:8]:
```suggestion
**`scrollbar-color`**  [CSS](/zh-CN/docs/Web/CSS) 属性设置滚动条轨道和滑块的颜色
```

File: files/zh-cn/web/css/scrollbar-color/index.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ slug: Web/CSS/scrollbar-color
...
 
-**track（轨道）**是指滚动条，其一般是固定的而不管滚动位置的背景。
+**track（轨道）** 是指滚动条，其一般是固定的而不管滚动位置的背景。
```
Comment (yin1999) [inline files/zh-cn/web/css/scrollbar-color/index.md:10]: [no space](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC) :)

```suggestion
**轨道**（track）是指滚动条，其一般是固定的而不管滚动位置的背景。
```

File: files/zh-cn/web/css/scrollbar-color/index.md [hunk 3]
```diff
@@ -5,9 +5,9 @@ slug: Web/CSS/scrollbar-color
...
 
 **thumb**（**拇指**）是指滚动条通常漂浮在轨道的顶部上的移动部分。
```
Comment (yin1999) [inline files/zh-cn/web/css/scrollbar-color/index.md:12]:
```suggestion
**滑块**（thumb）是指滚动条通常漂浮在轨道的顶部上的移动部分。
```

**Review by yin1999 — COMMENTED** [review]
Thanks for your contribution. Here are my suggestions:

File: files/zh-cn/web/css/scrollbar-color/index.md [hunk 4]
```diff
@@ -3,13 +3,13 @@ title: scrollbar-color
 slug: Web/CSS/scrollbar-color
 ---
 
-{{CSSRef}}{{SeeCompatTable}}
+{{CSSRef}}
 
-**`scrollbar-color`**该[CSS](/zh-CN/docs/Web/CSS)属性设置滚动条轨道和拇指的颜色
+**`scrollbar-color`**  [CSS](/zh-CN/docs/Web/CSS) 属性设置滚动条轨道和滑块的颜色
```
Comment (yin1999) [inline files/zh-cn/web/css/scrollbar-color/index.md:8]:
```suggestion
**`scrollbar-color`** [CSS](/zh-CN/docs/Web/CSS) 属性设置滚动条轨道和滑块的颜色
```

**Review by yin1999 — APPROVED** [review]
Thank you @kiner-tang. Welcome to the commmunity :tada:

**Comment by kiner-tang** [comment]
> Thank you @kiner-tang. Welcome to the commmunity 🎉

I would love to contribute to this project on an ongoing basis.

**Pull request merged**